jsonrpc/eth.go

```diff
@@ -624,14 +627,30 @@ func (e *Eth) relayTxToSequencerNode(input string) (interface{}, rpcError) {
 	return txHash, nil
 }
 
-func (e *Eth) tryToAddTxToPool(input string) (interface{}, rpcError) {
+func (e *Eth) tryToAddTxToPool(input string, dbTx pgx.Tx) (interface{}, rpcError) {
 	tx, err := hexToTx(input)
 	if err != nil {
 		return rpcErrorResponse(invalidParamsErrorCode, "invalid tx input", err)
 	}
+	ctx := context.Background()
+	procResp, err := e.state.ProcessTx(ctx, *tx, dbTx)
+	if err != nil {
```

Comment:
> We should return error for txs that won't be processed ever (nonce too low, ...)

For nonce is too low we have validateTx check, which returns an error in that case

> It could be interesting to add the tx as failed to the pool if the execution fails

Added